modules/ROOT/partials/module-loading/bundling-rollup-es6-zip_rollup.config.adoc

```diff
@@ -3,27 +3,27 @@
 Example `rollup.config.js`
 +
 ```js
```

Comment:
okay theres a few conversions todo, lets find and replace :)

Reply:
I should have done it for all files but I'll do another check

Reply:
Ok, now it seems all fixed